extensions/kafka/src/main/java/io/deephaven/kafka/publish/GenericRecordKeyOrValueSerializer.java

```diff
@@ -295,8 +295,10 @@ private static GenericRecordFieldProcessor makeInstantToMillisFieldProcessor(
         return makeGenericFieldProcessor(
                 fieldName,
                 chunkSource,
-                (final int ii, final ObjectChunk<?, Values> inputChunk) -> ((Instant) inputChunk.get(ii))
-                        .toEpochMilli());
+                (final int ii, final ObjectChunk<?, Values> inputChunk) -> {
+                    Instant val = (Instant) inputChunk.get(ii);
+                    return val == null ? null : val.toEpochMilli();
+                });
```

Comment:
We explicitly want to return `null : Long` as opposed to `NULL_LONG`?

Otherwise, `io.deephaven.time.DateTimeUtils#epochMillis(java.time.Instant)`.